pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -310,9 +321,12 @@ def available_devices(cls, **kwargs: Any) -> List[BackendInfo]:
             else:
                 provider = IBMProvider()
 
-        backend_info_list = [
-            cls._get_backend_info(backend) for backend in provider.backends()
-        ]
+        backend_info_list = []
+        for backend in provider.backends():
+            config = backend.configuration()
+            props = backend.properties()
```

Comment:
Have you checked if the properties you need at this point are not in the config?

Reply:
Yes I did try initially to just use the config. My reasoning that the properties should be passed also went as follows... `_get_backend_info` calls `process_characterisation_from_config`, which does use the properties if available. The currently published code (before this PR) does seem to avail itself of properties where it exists because the first thing that `process_characterisation` does is call
```properties = cast("BackendProperties", backend.properties())```

Reply:
Ah okay! Sounds good